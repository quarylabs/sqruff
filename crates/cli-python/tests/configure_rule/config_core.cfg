[sqruff]
dialect = ansi
rules = core

