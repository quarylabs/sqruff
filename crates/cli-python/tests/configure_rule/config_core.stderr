== [tests/configure_rule/_example.sql] FAIL
L:   2 | P:   1 | LT12 | Files must end with a single trailing newline.
                       | [layout.end_of_file]
The linter processed 1 file(s).
All Finished
